lib/matplotlib/streamplot.py

```diff
@@ -52,6 +52,8 @@ def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
         See :class:`~matplotlib.patches.FancyArrowPatch`.
     *minlength* : float
         Minimum length of streamline in axes coordinates.
+    *start_points*: Nx2 array
+        Coordinates of starting points for the streamlines. In array coordinates.
```

Comment:
What do you mean by "array coordinates"?  I would expect starting points to be specified in data coordinates--that is, in the same coordinates as your x and y input arrays.